### DISCUSSION TITLE
Welcome to SpartanProtocol-DAppV2 Discussions!

### DISCUSSION THREAD

**Discussion by SamusElderg** [spartan-protocol/SpartanProtocol-DAppV2#319] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by f1ntch** [comment]
Hi there community! I am f1ntch and I have built the UI together with Samus for dAppv1.1 and dAppv2. I mainly fix UI issues before and after the launch and help to add new features based on community input.

It is by listening to the community that we can solve issues and improve our dApp. So any idea big or small is welcome. Feel free to reach out and to contribute any help is much appreciated! 🙏

**Reply by SamusElderg** [reply]
welcome f1ntch! Appreciate all your contributions and look forward to continuing the BUIDl with you ongoing!
